scripts/ci/macos/repeat_hdiutil.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+((${BASH_VERSION%%.*} >= 4)) || { echo >&2 "$0: Error: Please upgrade Bash."; exit 1; }
+
+set -euxo pipefail
+
+cmd_string=/usr/bin/hdiutil "$@" -debug
+
+max_retry=5
+
+counter=0
+
+until cmd_string
+do
+   sleep 1
+   [[ counter -eq $max_retry ]] && echo "CPack failed despite retry attempts!" && exit 1
```

Comment:
And maybe here?..

Reply:
```suggestion
   if [[ $counter -eq $max_retry ]]; then
       echo "CPack failed despite retry attempts!"
       exit 1
   fi
```
Also I'd prefer something like this in case `echo` fails, and also to avoid failing with `-e`